CKAN 2.9 file type labels

Alexandra-Green opened the pull request:
## What this PR accomplishes

Update the file type labels

## Issue(s) addressed

- update the color of file type labels and remove the pseudo 3D styling to keep them as plain rectangles with rounded corners 
- Font color #000, font weight: 600

## What needs review

- File type labels are in the correct colour and font weight on the dataset search and individual dataset/ministry/group page

### Notes:

- Padding and font sizes/font weight are different on the Figma (I added the Figma to the ticket for visual color reference) but the ticket only asks for the color and font weight changed. This may be something to add to dataset search page, if needed. As this PR only reflects the ticket.

KatiRG commented on ckanext/ontario_theme/fanstatic/external/ontario_theme.css:
```diff
@@ -483,72 +483,11 @@ div.card p {
-  height: 35px;
-  background-position: -414px -62px;
+.label {
+  color: #FFFFFF;
```
why is this needed?

KatiRG commented:
New stylings to apply:
   - make font weight semi-bold
   - make font-size smaller (15px)
   - make rectangle smaller:
   
![image](https://user-images.githubusercontent.com/1254764/222230634-3817f193-b62b-41d5-8bce-eaca70b40257.png)

   - add 16px spacing between labels:
   
![image](https://user-images.githubusercontent.com/1254764/222230758-cdd382e4-3c4f-4822-8988-4be8b3c4ed3a.png)

Alexandra-Green commented:
I've added the `0.4em 1em` padding, added `16px` left margin to the file type labels and the font size is now 15px and semi-bold (`font-weight: 600`).

KatiRG reviewed (changes requested):
The linear gradients are still being applied to the label backgrounds:
![Screen Shot 2023-03-14 at 09 46 56](https://user-images.githubusercontent.com/1254764/225021266-fd4a590b-9089-4361-8118-bbf0beb8934c.png)

Alexandra-Green commented:
> The linear gradients are still being applied to the label backgrounds:

Those aren't even the right colours, so that is weird. I'll take a look

KatiRG reviewed (approved):
@Alexandra-Green  Sorry, must have been a cache thing!

Can you merge the upgrade branch? Otherwise good to go. Thanks!
